app/Jobs/TileSingleImage.php

```diff
@@ -50,7 +54,8 @@ class TileSingleImage extends Job implements ShouldQueue
     public function __construct(Image $image)
     {
         $this->image = $image;
-        $this->tempPath = config('image.tiles.tmp_dir')."/{$image->uuid}";
+        $fragment = fragment_uuid_path($image->uuid);
+        $this->tempPath = config('image.tiles.tmp_dir') . "/{$fragment}";
```

Comment:
Ah now I see this. This will clutter the temp directory, as a new tempPath is generated like this: `/tmp/ba/da/bada55-uuid/` but `deleteDirectory` below will only delete the last `bada55-uuid`. At some point you end up with possibly 256² empty directories.

The "fragment" stuff is only needed at the target storage location. For temporary storage here the UUID is totally fine.